R4ML/tests/testthat/test-util.io.R

```diff
@@ -39,6 +39,7 @@ test_that("r4ml.read.csv", {
       tmp_file <- paste0(tempfile(), ".csv")
       write.table(iris, file = tmp_file, sep = seperators[i], row.names = FALSE)
       df <- r4ml.read.csv(tmp_file, header = TRUE, sep = seperators[i])
+      df <- SparkR::as.data.frame(df)
```

Comment:
The unit test case related to r4ml.read.csv was failing with the error saying "**comparison (1) is possible only for atomic and list types**"
So converting the r4ml.frame object to data.frame at L42 before comparing with iris using all method at L45.